firewood/src/merkle.rs

```diff
@@ -796,10 +807,33 @@ impl<'a, T: PartialEq> PrefixOverlap<'a, T> {
 #[cfg(test)]
 #[allow(clippy::indexing_slicing, clippy::unwrap_used)]
 mod tests {
+    use std::time::{SystemTime, UNIX_EPOCH};
+
     use super::*;
+    use rand::{rngs::StdRng, Rng, SeedableRng};
     use storage::MemStore;
     use test_case::test_case;
 
+    // Returns n random key-value pairs.
+    fn generate_random_kvs(seed: u64, n: usize) -> Vec<(Vec<u8>, Vec<u8>)> {
+        println!("Used seed: {}", seed);
```

Comment:
nit: `println!` -> `eprintln!` if you need this

Reply:
Done